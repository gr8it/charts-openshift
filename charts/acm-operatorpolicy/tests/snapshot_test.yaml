suite: Snapshot test
templates:
  - templates/*
values:
  - ../values.example.yaml
release:
  name: acm-operatorpolicy
  namespace: operator-namespace

tests:
  - it: manifests should match snapshot
    asserts:
      - matchSnapshot: {}
