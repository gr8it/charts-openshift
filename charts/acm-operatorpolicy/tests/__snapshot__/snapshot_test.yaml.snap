manifests should match snapshot:
  1: |
    apiVersion: policy.open-cluster-management.io/v1beta1
    kind: OperatorPolicy
    metadata:
      annotations: {}
      labels:
        app.kubernetes.io/instance: acm-operatorpolicy
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: acm-operatorpolicy
        app.kubernetes.io/version: v1beta1
        helm.sh/chart: acm-operatorpolicy-1.4.0
      name: acm-operatorpolicy
      namespace: open-cluster-management-policies
    spec:
      complianceConfig:
        catalogSourceUnhealthy: Compliant
        deploymentsUnavailable: NonCompliant
        upgradesAvailable: Compliant
      complianceType: musthave
      remediationAction: enforce
      removalBehavior:
        clusterServiceVersions: Delete
        customResourceDefinitions: Keep
        operatorGroups: DeleteIfUnused
        subscriptions: Delete
      severity: medium
      subscription:
        channel: stable-v1
        config:
          env:
            - name: HTTP_PROXY
              value: proxy.example.com
            - name: HTTPS_PROXY
              value: proxy.example.com
            - name: NO_PROXY
              value: localhost,127.0.0.1,.svc,.cluster.local,.example.com
        name: openshift-cert-manager-operator
        source: redhat-operators
        sourceNamespace: openshift-marketplace
        startingCSV: cert-manager-operator.v1.15.1
      upgradeApproval: Automatic
      versions:
        - cert-manager-operator.v1.15.1
